### PR TITLE
Added a positive nr field to the form fields

### PR DESCRIPTION
**Pull request by sewagodimo**

### CODE REVIEW

File: molo/surveys/tests/test_rules.py [hunk 1]
```diff
@@ -57,13 +57,14 @@ def setUp(self):
...
 
-        # Create survey submission
+    # Create survey submission
         data = {
             self.singleline_text.clean_name: 'super random text',
             self.checkboxes.clean_name: ['choice 3', 'choice 1'],
             self.checkbox.clean_name: True,
             self.number.clean_name: 5
         }
+
```
Comment (sewagodimo) [inline molo/surveys/tests/test_rules.py]: @miltontony  you added these. Should I remove them or keep them?

Comment (miltontony) [reply molo/surveys/tests/test_rules.py]: You can remove it

File: molo/surveys/tests/test_rules.py [hunk 2]
```diff
@@ -184,6 +184,38 @@ def test_not_logged_in_user_fails(self):
         self.assertFalse(rule.test_user(self.request))
 
 
+@pytest.mark.django_db
+class TestSurveyNegativeNumberDataRuleSegmentation(
```
Comment (KaitCrawford) [inline molo/surveys/tests/test_rules.py]: I don't think this is the correct place for these tests (or the correct name). The tests in this file are all to test the code in `rules.py` but you didn't or change any rules. I'd guess you need to find out where the forms are tested and add the test there.

File: molo/surveys/tests/test_rules.py [hunk 3]
```diff
@@ -184,6 +184,38 @@ def test_not_logged_in_user_fails(self):
...
+        self.request = self.request_factory.get('/')
+        self.request.user = get_user_model().objects.create_user(
+            username='tester', email='tester@example.com', password='tester')
+
+    def test_negative_number_validation(self):
+        # Create survey
+        self.survey = PersonalisableSurvey(title='Test Survey')
+        SurveysIndexPage.objects.first().add_child(instance=self.survey)
+
+        self.neg_number = PersonalisableSurveyFormField.objects.create(
```
Comment (KaitCrawford) [inline molo/surveys/tests/test_rules.py]: You're currently only testing the field for a PersonalisableSurvey, you probably need to add a similar test to check this works for a normal survey (MoloSurvey)

File: molo/surveys/tests/test_rules.py [hunk 4]
```diff
@@ -170,6 +175,22 @@ def test_failing_number_rule(self):
...
+            survey=self.survey, operator=SurveySubmissionDataRule.CONTAINS,
+            expected_response='8',
+            field_name=self.positive_number.clean_name)
+
+        self.assertTrue(rule.test_user(self.request))
+
+    def test_failing_positive_number_rule(self):
+        rule = SurveySubmissionDataRule(
+            survey=self.survey, operator=SurveySubmissionDataRule.CONTAINS,
+            expected_response='-8',
```
Comment (KaitCrawford) [inline molo/surveys/tests/test_rules.py]: Please change this to a valid response (eg 4)

**Comment by KaitCrawford** [comment]
:+1:

**Pull request merged**